src/app/features/recipes/recipes-listing-filter/recipes-listing-filter.component.ts

```diff
@@ -26,8 +33,7 @@ export class RecipesListingFilterComponent implements OnInit {
   }
   toggle(index) {
```

Comment:
@SarahSamy rename this function to something more descriptive for example `toggleExpand`

Reply:
OK, it will be changed.